### PR TITLE
Rename LSHIFT

### PR DESCRIPTION
**Pull request by stevechang0459**
Rename LSHIFT as BITLSHIFT
Add BITRSHIFT

### CODE REVIEW

File: src/main.h [hunk 1]
```diff
@@ -2,25 +2,25 @@
...
+        | BITLSHIFT(1, ERROR) \
+        | BITLSHIFT(1, WARN) \
+        | BITLSHIFT(1, DEBUG) \
+        | BITLSHIFT(1, INFO) \
+        | BITLSHIFT(1, INIT) \
         )
 
 #if 1
-extern const char *main_trace_header[];
+// extern const char *main_trace_header[];
```
Comment (s980540) [inline src/main.h]: Remove unnecessary code/comment like main_trace_header.

File: src/smbus/smbus.h [hunk 2]
```diff
@@ -33,25 +33,25 @@
...
-        | LSHIFT(INIT) \
+        | BITLSHIFT(1, ERROR) \
+        | BITLSHIFT(1, WARN) \
+        | BITLSHIFT(1, DEBUG) \
+        | BITLSHIFT(1, INFO) \
+        | BITLSHIFT(1, INIT) \
         )
 
 #if 1
 extern const char *smbus_trace_header[];
```
Comment (s980540) [inline src/smbus/smbus.h:44]: Remove unnecessary code/comment.

File: src/mctp/mctp.h [hunk 3]
```diff
@@ -31,25 +31,25 @@
...
-        | LSHIFT(INIT) \
+        | BITLSHIFT(1, ERROR) \
+        | BITLSHIFT(1, WARN) \
+        | BITLSHIFT(1, DEBUG) \
+        | BITLSHIFT(1, INFO) \
+        | BITLSHIFT(1, INIT) \
         )
 
 #if 1
 extern const char *mctp_trace_header[];
```
Comment (s980540) [inline src/mctp/mctp.h:42]: Remove unnecessary code/comment.

**Review by s980540 — CHANGES_REQUESTED** [review]
Just clean up the code than you can commit again.

File: src/main.h (hunk 1 above)
Comment (stevechang0459) [reply src/main.h]: OK

**Review by s980540 — APPROVED** [review]

File: src/mctp/mctp.h (hunk 3 above)
Comment (s980540) [reply src/mctp/mctp.h:42]: Since different C files use mctp_trace, we cannot remove mctp_trace_header here.

File: src/smbus/smbus.h (hunk 2 above)
Comment (s980540) [reply src/smbus/smbus.h:44]: Since different C files use smbus_trace, we cannot remove smbus_trace_header here.

File: src/main.h (hunk 1 above)
Comment (stevechang0459) [reply src/main.h]: I had just fixed it and push the new commit.

File: src/mctp/mctp.h (hunk 3 above)
Comment (stevechang0459) [reply src/mctp/mctp.h:42]: Since different C files use mctp_trace, we cannot remove mctp_trace_header here.

File: src/smbus/smbus.h (hunk 2 above)
Comment (stevechang0459) [reply src/smbus/smbus.h:44]: Since different C files use smbus_trace, we cannot remove smbus_trace_header here.

**Pull request merged**